main/zoe/api/zoe-helpers.md

```diff
@@ -202,6 +202,53 @@ import {
 swap(zcf, firstSeat, secondSeat);
 ```
 
+## swapExact(zcf, leftSeat, rightSeat, leftHasExitedMsg, rightHasExitedMsg)
+- `zcf` `{ContractFacet}`
+- `leftSeat` `{ZCFSeat}`
+- `rightSeat` `{ZCFSeat}`
+- `leftHasExitedMsg` - `{String}` - Optional
+- `rightHasExitedMsg` - `{String}` - Optional
+- Returns: `defaultAcceptanceMsg`
+
+**Note**: `swapExact()` uses the `swap()` method which is a specific use of `trade()`. In `swap(),` 
+for both `seats`, everything a `seat` wants is given to it, having been
+taken from the other `seat`. `swap()` exits both `seats`, but `trade()` does not.
+- Use `trade()` when any of these are true:
+  - The `seats` have different keywords.
+  - The `amounts` to be reallocated don't exactly match the wants of the `seats`. 
+  - You want to continue interacting with the `seats` after the trade.
```

Comment:
It does not use swap! It uses `trade`.

Reply:
Yeah, I misphrased there. How about:

**Note**: `swapExact()` and `swap()` are specific uses of `trade()`. In `swap()` and `swapExact()`,
for both `seats`, everything a `seat` wants is given to it, having been
taken from the other `seat`. `swap()` and `swapExact()` exit both `seats`, but `trade()` does not.

Reply:
Sounds good